src/test/java/ch/zhaw/robopatrol/res/RouteTest.java

```diff
@@ -99,6 +121,123 @@ public void testDelete() {
         assertThat(points, hasSize(0));
     }
 
+    @Test
+    public void testEquals(){
+      Waypoint p1 = point();
+      Waypoint p2 = point();
+
+      assertThat(p1.equals(p2), is(true));
```

Comment:
`assertThat(p1, is(p2))`

This does the same thing, but provides a better error message in case of failure.